setup/src/Magento/Setup/Module/Di/Code/Reader/InvalidFileException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Magento\Setup\Module\Di\Code\Reader;
+
+class InvalidFileException extends \InvalidArgumentException
```

Comment:
Custom Exceptions are not recommended to be introduced in each module.

Here is more discussion on this subject -  https://github.com/magento/magento2/pull/9315/files#r112514206